tests/server/test_environment_metrics.py

```diff
@@ -1406,7 +1406,8 @@ async def test_metric_aggregation_no_date(
     assert result.result["data"]["metrics"]["gauge_metric1"] == [None for _ in range(10)]
 
 
-async def test_cleanup_environment_metrics(init_dataclasses_and_load_schema) -> None:
+@pytest.mark.parametrize("env1_halted, env2_halted", [(True, True), (False, True), (True, False), (False, False)])
```

Comment:
If you want to do all possible combinations you can do it this way:
```suggestion
@pytest.mark.parametrize("env1_halted", [True, False])
@pytest.mark.parametrize("env2_halted", [True, False])
```

Reply:
ow that's easier :p